jsx/Form.js

```diff
@@ -22,14 +21,41 @@ ButtonElement, LorisElement
  * Note that if both are passed `this.props.formElements` is displayed first.
  *
  */
-class FormElement extends React.Component {
-  constructor() {
-    super();
-    this.getFormElements = this.getFormElements.bind(this);
-    this.handleSubmit = this.handleSubmit.bind(this);
-  }
+var FormElement = React.createClass({
```

Comment:
it looks like you were based on an old branch, the Form components have changed a lot since then
adding Needs Rebase